ttg/ttg/parsec/ttg.h

```diff
@@ -68,6 +68,7 @@ int parsec_add_fetch_runtime_task(parsec_taskpool_t *tp, int tasks);
 
 namespace ttg_parsec {
   static __thread parsec_task_t *parsec_ttg_caller;
+  static __thread parsec_execution_stream_t *parsec_ttg_es;
```

Comment:
- `__thread` is nonstandard, switch to `thread_local`: http://en.cppreference.com/w/cpp/language/storage_duration
- probably do not need `static` to indicate internal linkage here? `static` will make different variables in different object files that include `parsec/ttg.h`. Presumably these should just be per thread, not per thread + per object file. So I suggest `inline thread_local` (see https://en.cppreference.com/w/cpp/language/inline)